js/cfi_generator.js

```diff
@@ -193,10 +193,12 @@ var obj = {
         this.validatePackageDocument(packageDocument, contentDocumentName);
 
         // Get the start node (itemref element) that references the content document
-        $itemRefStartNode = $("itemref[idref='" + contentDocumentName + "']", $(packageDocument));
+        var $itemRefStartNode = $($(packageDocument.getElementsByTagNameNS('*', 'itemref')).toArray().find(function (node) {
+            return node.getAttribute('idref') === contentDocumentName;
+        }));
```

Comment:
Note that jQuery's `toArray()` return a standard ECMAScript array: https://api.jquery.com/toArray/